subscription/src/main/java/org/killbill/billing/subscription/api/user/DefaultSubscriptionBase.java

```diff
@@ -460,14 +460,26 @@ public ProductCategory getCategory() {
 
     @Override
     public Integer getBillCycleDayLocal() {
+        return getQuantityOrBCD(true);
+    }
+
+    @Override
+    public Integer getQuantity() {
+        return getQuantityOrBCD(false);
+    }
+
+
+    private Integer getQuantityOrBCD(final boolean bcd) {
```

Comment:
Nit: I find the method a tad confusing.

Maybe `getNextValueForTransition(final SubscriptionBaseTransitionType transitionType)`?

Reply:
Fixed in https://github.com/killbill/killbill/pull/1805/commits/9e05be1dc910b224d8bc28994c7d43a909225277